### PR TITLE
Advertise jax support a bit more

### PR DESCRIPTION
**Pull request by alexfikl**
2c6414c2e87b00a7d2a472f9791bf20b0231d7d3 is a bit scary looking, but it just moves stuff around and adds a few vim markers to match the other array contexts.

### CODE REVIEW

File: doc/array_context.rst [hunk 1]
```diff
@@ -20,7 +27,7 @@ Lazy/Deferred evaluation array context based on :mod:`pytato`
 .. automodule:: arraycontext.impl.pytato
 
 
-Array context :mod:`jax.numpy`
+Array context based on :mod:`jax.numpy`
 -------------------------------------------------------------
```
Comment (kaushikcfd) [inline doc/array_context.rst:31]:
```suggestion
---------------------------------------
```

**Review by kaushikcfd — APPROVED** [review]
Thanks, LGTM!

File: doc/array_context.rst (hunk 1 above)
Comment (alexfikl) [reply doc/array_context.rst]: Nice catch! Fixed in https://github.com/inducer/arraycontext/compare/b12875d4eac2b19d71f4a8495fc2ff136ce77dad..2983116866fe389e7b923cada1b79715b69bf4b1.

Comment (kaushikcfd) [reply doc/array_context.rst]: Thanks!

**Comment by inducer** [comment]
Thanks!

**Pull request merged**